refapp/SKWebApi/Model/AskResult.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace SemanticKernel.Service.Model;
+
+public class AskResult
```

Comment:
Should this class and its members be renamed? @craigomatic

Reply:
See above.